.github/workflows/test_chart.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: kind load docker-image ${{ env.MANIFEST_IMG }}:${{ env.TAG }}
 
       - name: Run chart-testing (install)
-        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --set cluster-api-operator.cert-manager.enabled=true
+        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --set cluster-api-operator.cert-manager.enabled=true --set clusterAPI.enabled=false
```

Comment:
A bit unrelated to this PR, but why do we need to add capi operator chart repor [there](https://github.com/rancher-sandbox/rancher-turtles/pull/109/files#diff-6e5e920e8ceef9fedf6bc5cd6ac8e0774002e0be57cb1f15b8860a66d64570e7R45)? I think this job will work without it.

Reply:
Tried that, see the error from chart-tester, resolved.